requirements/edx/development.txt

```diff
@@ -1,7 +1,7 @@
 #
 # Dependencies that are used in development only - and are *NOT* needed to be installed in staging/production.
 #
-# These must be installed manually in your development environment using:
+# These are installed automatically in devstack, and can also be installed manually using:
```

Comment:
Both appear to use roles/edxapp/vars/devstack.yml which should include this file.